packages/flutter_tools/lib/src/localizations/localizations_utils.dart

```diff
@@ -165,7 +163,7 @@ Map<String, List<String>> _parseSection(String section) {
     final String name = line.substring(0, colon);
     final String value = line.substring(colon + 2);
     lastHeading = result.putIfAbsent(name, () => <String>[]);
-    result[name].add(value);
+    result[name]!.add(value);
```

Comment:
I'm wondering if this makes more sense to get rid of the null check:
```dart
    lastHeading = result.putIfAbsent(name, () => <String>[value]);
```

Reply:
Wouldn't that add it twice to the list on the first insertion?

Reply:
Oh, sorry, I mean remove this line and just keep everything on the same line above.